aws/route53/test_route53_zones.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from aws.route53.resources import zones, cnames
+
+
+@pytest.mark.parametrize(
+    "cnames", cnames(),
```

Comment:
You should pass some `ids` parameter here, since otherwise failing tests are only identified by something like "cnames-123", which isn't really helpful.